src/quo2/foundations/colors.cljs

```diff
@@ -246,3 +247,7 @@
 (defn dark?
   []
   (theme/dark?))
+
+;; Turquoise
+(def turquoise-50-opa-40 (custom-color :turquoise 50 40))
+(def turquoise-60 (get-in colors-map [:turquoise 60]))
```

Comment:
Please can you elaborate, why they are needed?

Reply:
@Parveshdhull, @mattgle added them because of the discussion in this thread https://github.com/status-im/status-mobile/pull/14652#discussion_r1058285737

It was a misdirection on my part, sorry about that.

@mattgle, you can just use:

```clojure
(custom-color :turquoise 50 40)
```

Reply:
Perfect guys! Will do the changes!